### PR TITLE
fix date format causing runtime error when new task added

### PR DESCRIPTION
**Pull request by anserwaseem**

### CODE REVIEW

File: src/renderer/components/TaskItem/index.tsx [hunk 1]
```diff
@@ -45,9 +45,7 @@ export default function TaskItem({
       />
       <div className="col">
         <p className={styles['task-label']}>{label}</p>
-        <p className={styles['task-date']}>
-          {format(new Date(date), "E., dd 'de' MMM")}
-        </p>
+        <p className={styles['task-date']}>{new Date(date).toJSON()}</p>
```
Comment (tuliocll) [inline src/renderer/components/TaskItem/index.tsx]: We can keep the format here.

File: src/renderer/components/Modal/index.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default function Modal({ onClose, initialData, onSave }: Modal) {
 
     onSave({
       title,
-      date: initialData?.date || new Date().toLocaleString(),
+      date: initialData?.date || new Date().toJSON(),
```
Comment (tuliocll) [inline src/renderer/components/Modal/index.tsx:21]: Sorry, I did see that I use `toLocaleString()` and not `toDateString()`.

**Review by tuliocll — CHANGES_REQUESTED** [review]

**Comment by tuliocll** [comment]
Hi @anserwaseem, sorry about that. Thank you for the fix!
Fix the `TaskItem` file so I can merge the PR.

File: src/renderer/components/TaskItem/index.tsx (hunk 1 above)
Comment (anserwaseem) [reply src/renderer/components/TaskItem/index.tsx]: this issue not reproducible anymore so I think we should not merge this code as of now, wdy think?

Comment (tuliocll) [reply src/renderer/components/TaskItem/index.tsx]: sure, you can revert this part, only the fix on `onSave()` solves the problem, we can keep the `format` function here.

**Comment by tuliocll** [comment]
Thank you 🎉

**Pull request merged**